ara/cli/stats.py

```diff
@@ -17,19 +17,30 @@
 
 from cliff.lister import Lister
 from cliff.show import ShowOne
-from ara import models, utils
+from ara import models
+from ara.fields import Field
 
-FIELDS = (
-    ('ID',),
-    ('Host', 'host.name'),
-    ('Playbook',),
-    ('Changed',),
-    ('Failed',),
-    ('Ok',),
-    ('Skipped',),
-    ('Unreachable',),
+COMMON_FIELDS = (
+    Field('Changed'),
+    Field('Failed'),
+    Field('Ok'),
+    Field('Skipped'),
+    Field('Unreachable'),
 )
 
+LIST_FIELDS = (
```

Comment:
Because order matters, and `COMMON_FILES` is appended, while we want `ID` and `Host` at the top of the list.